charts/incubator/hyperswitch-app/configs/misc.toml

```diff
@@ -1,174 +1,207 @@
+sampling_rate = {{ .Values.application.server.log.telemetry.sampling_rate }}                                   # decimal rate between 0.0 - 1.0
+otel_exporter_otlp_endpoint = "{{ .Values.application.server.log.telemetry.otel_exporter_otlp_endpoint }}" # endpoint to send metrics and traces to, can include port number
+otel_exporter_otlp_timeout = {{ .Values.application.server.log.telemetry.otel_exporter_otlp_timeout }}                     # timeout (in milliseconds) for sending metrics and traces
+use_xray_generator = {{ .Values.application.server.log.telemetry.use_xray_generator }}                            # Set this to true for AWS X-ray compatible traces
+route_to_trace = {{ .Values.application.server.log.telemetry.route_to_trace }}
+bg_metrics_collection_interval_in_secs = {{ .Values.application.server.log.telemetry.bg_metrics_collection_interval_in_secs }}           # Interval for collecting the metrics in background thread
 
 [lock_settings]
-delay_between_retries_in_milliseconds = 500 # Delay between retries in milliseconds
-redis_lock_expiry_seconds = 180             # Seconds before the redis lock expires
+delay_between_retries_in_milliseconds = {{ .Values.application.server.lock_settings.delay_between_retries_in_milliseconds }} # Delay between retries in milliseconds
+redis_lock_expiry_seconds = {{ .Values.application.server.lock_settings.redis_lock_expiry_seconds }}             # Seconds before the redis lock expires
 
 [master_database]
-pool_size = "20"
-connection_timeout = 10   
-queue_strategy = "Fifo"
+pool_size = "{{ .Values.application.server.master_database.pool_size }}"
+connection_timeout = {{ .Values.application.server.master_database.connection_timeout }}   
+queue_strategy = "{{ .Values.application.server.master_database.queue_strategy }}"
 
 [generic_link]
+[generic_link.payment_method_collect]
+sdk_url = {{ include "hyperswitchWeb.hyperloaderUrl" . | quote }}
+expiry = 900
+
 [generic_link.payment_method_collect.ui_config]
-theme = "#4285F4"
-logo = "https://app.hyperswitch.io/HyperswitchFavicon.png"
-merchant_name = "HyperSwitch"
+theme = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.theme }}"
+logo = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.logo }}"
+merchant_name = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.merchant_name }}"
+
 [generic_link.payment_method_collect.enabled_payment_methods]
-card = "credit,debit"
-bank_transfer = "ach,bacs,sepa"
-wallet = "paypal,pix,venmo"
+card = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.card }}"
+bank_transfer = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.bank_transfer }}"
+wallet = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.wallet }}"
```

Comment:
why is this need to be from values.yaml? isnt this a feature we provide?

Reply:
Adding it in values.yaml, just provides control to the users who deploy our services by themselves, if they want to disable some features they can easily change it and redeploy instead of changing it in the actual file. Also readme docs will be easier to generate if we add it in values.yaml
Check [this readme](https://github.com/juspay/hyperswitch-helm/blob/8b456391c4378976a7015a3d6b3853652068fb6c/charts/incubator/hyperswitch-stack/README.md) this is autogenerated from values.yaml